config/locales/en.yml

```diff
@@ -112,6 +112,11 @@ en:
     mps_provided_2: Service is provided by
     mps: Money & Pensions Service
 
+  covid_banner: 
+    head: Coronavirus – what it means for you
+    body_html: <a href="%{href}">Find out what you're entitled to</a>
+    href: /en/articles/coronavirus-what-it-means-for-you
```

Comment:
This url needs to be updated to point to the website. Same with the Welsh file.

Reply:
@andreferreiradlw OK, I have updated those.